mlflow/pyfunc/__init__.py

```diff
@@ -1108,6 +1108,7 @@ def spark_udf(
     result_type=None,
     env_manager=_EnvManager.LOCAL,
     params: Optional[Dict[str, Any]] = None,
+    extra_env_vars: Optional[Dict[str, str]] = None,
```

Comment:
I think we can simplify name to `extra_env`, like `subprocess.Popen` 's `env` argument.